### PR TITLE
LPD-17954 Fix CalendarConfiguration#ViewEventDetailsAllDayWithModifiedUserTimeZone

### PR DESCRIPTION
**Pull request by larissacribeiro**
Related Issue: [LPD-17954](https://liferay.atlassian.net/browse/LPD-17954)

### CODE REVIEW

File: modules/apps/calendar/calendar-web-test/src/testFunctional/tests/CalendarConfiguration.testcase [hunk 1]
```diff
@@ -346,9 +346,6 @@ definition {
 		var endDateTime = CalendarEvent.getCalendarTimeHour(status = "End");
 		var startDateTime = CalendarEvent.getCalendarTimeHour(status = "Start");
 
-		var startDateTime = CalendarEvent.format12To24Hours(time12Hours = ${startDateTime});
-		var endDateTime = CalendarEvent.format12To24Hours(time12Hours = ${endDateTime});
-
```
Comment (igor-franca) [inline modules/apps/calendar/calendar-web-test/src/testFunctional/tests/CalendarConfiguration.testcase:351]: Hi @larissacribeiro , great work with this PR!

**Review by igor-franca — APPROVED** [review]

**Comment by igor-franca** [comment]
ci:test:calendar

**Comment by larissacribeiro** [comment]
The `CalendarConfiguration#ViewEventDetailsAllDayWithModifiedUserTimeZone` [test](https://testray.liferay.com/home/-/testray/case_results/2542777774?redirect=https%3A%2F%2Ftestray.liferay.com%2Fhome%2F-%2Ftestray%2Fcase_results%2Findex%3FtestrayBuildId%3D2542777524%26testrayRunId%3D2542777602%26cur%3D1%26delta%3D200%26orderByCol%3Dstatus_sortable%26orderByType%3Dasc%26resetCur%3Dfalse) is not failing anymore

**Comment by igor-franca** [comment]
ci:forward

**Pull request closed without merging**